typescript/test/unit/baseline.ts

```diff
@@ -14,18 +14,23 @@
 
 import * as assert from 'assert';
 import { execSync } from 'child_process';
-import * as fs from 'fs';
+import * as fs from 'fs-extra';
 import * as path from 'path';
 import * as rimraf from 'rimraf';
 
 const cwd = process.cwd();
+const BASELINE_EXTENSION = '.baseline';
 
 const OUTPUT_DIR = path.join(cwd, '.baseline-test-out');
-const GENERATED_CLIENT_FILE = path.join(
-  OUTPUT_DIR,
-  'src',
-  'v1beta1',
-  'echo_client.ts'
+const BASELINE_DIR = path.join(
+  __dirname,
+  '..',
+  '..',
+  '..',
+  'typescript',
+  'test',
+  'testdata',
+  'showcase'
```

Comment:
Let's remove `showcase` from here, then it will be very easy to add more full-sized baseline tests for other libraries.

Reply:
(I mean, the showcase test will start from `path.join(BASELINE_DIR, 'showcase')`, some other baseline test - the multiple protos one - can start from some other folder, etc., all sharing the same comparison code.

Reply:
Sure!